sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -1775,9 +1775,12 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 				getDao(EpiData.class).executeRaw("DROP TABLE tmp_epidata;");
 			case 247:
 				currentVersion = 247;
-
-				getDao(Contact.class).executeRaw("ALTER TABLE contacts ADD column multidaycontact boolean default false;");
-				getDao(Contact.class).executeRaw("ALTER TABLE contacts ADD column firstcontactdate timestamp;");
+				try {
+                    getDao(Contact.class).executeRaw("ALTER TABLE contacts ADD column multiDayContact boolean default false;");
+                    getDao(Contact.class).executeRaw("ALTER TABLE contacts ADD column firstContactDate timestamp;");
+                } catch (SQLException e) {
```

Comment:
Remove the try-catch block; the only reason this might already exist is that the state the app is in on the mobile phone is broken in the first place